Dokan/Dokan/Scenes/ProductDetails/ReviewsView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ReviewsView.swift
+//  Dokan
+//
+//  Created by Clara Mounir Adly on 16/07/2022.
+//
+
+import Foundation
+import UIKit
+
+class ReviewsView: UIView, UITableViewDelegate, UITableViewDataSource {
+    
+    var uiMODEL: ReviewsUiModel?
+   // MARK: outlets
+    @IBOutlet weak var contentView: UIView!
+    @IBOutlet weak var reviews: UILabel!
+    @IBOutlet weak var rating: UILabel!
+    @IBOutlet weak var customerReview: UITableView!
+    @IBOutlet weak var seeAllReviewsButton: UIButton!
+    
+    // MARK: functions
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        uiMODEL!.reviews.count
+    }
+    
+   // func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+ //       guard let cell = tableView.dequeueReusableCell(withIdentifier: " ") else { return // } 
+      
+    //    return cell
+  //  }
+    
+    override init(frame: CGRect) {
+          super.init(frame: frame)
+          commonInit()
+        
+        customerReview.dataSource = self
+        customerReview.delegate = self
+      }
+      required init?(coder aDecoder: NSCoder) {
+          super.init(coder: aDecoder)
+          commonInit()
+      }
+      func commonInit() {
+          Bundle.main.loadNibNamed("ReviewsView", owner: self, options: nil)
+          contentView.fixInView(self)
+      }
+  }
+
+  extension UIView
+  {
+      func fixInView(_ container: UIView!) -> Void{
```

Comment:
```suggestion
      func fixInView(_ container: UIView) -> Void {
```
No need to force unwrap the parameter here.